agent/lib/docker/network.rb

```diff
@@ -0,0 +1,18 @@
+require 'docker'
+
+module Docker
+  class Network
+
+    # Monkey patched temporarily, we should create a real PR
```

Comment:
What about this recent commit: https://github.com/swipely/docker-api/commit/0c95907eaf9d1d6ac8ac3cbe10989450ddd6d57f

It adds a `network.connect(container, {}, { endpointconfig: ... })`